dashboard/db/migrate/20180806214228_create_queued_account_purges.rb

```diff
@@ -0,0 +1,10 @@
+class CreateQueuedAccountPurges < ActiveRecord::Migration[5.0]
+  def change
+    create_table :queued_account_purges do |t|
```

Comment:
Note for reviewers: I believe this migration can go out along with the rest of the change because the only code depending on it runs within a cronjob, which will only run on daemon and only once per night - no timing issues expected.

Reply:
Even if new front end code that relied on it went out at the same time, the migration happens first so it would be ok. The problem we've gotten into is by changing a table that existing deployed code already depends on. Data migrations happen immediately and then it's another ~ 45 mins until the new front ends are ready and swapped in.